src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx

```diff
@@ -158,16 +158,16 @@ const RSSFeeds = connect<{}, SignUpForm>((props) => {
       setValidating(true);
       controllerRef?.current?.abort();
       controllerRef.current = new AbortController();
+      // Allow a list of URLs, separated by spaces
+      const urls = values.blogUrl.split(/ +/);
       const response = await fetch(`${feedDiscoveryServiceUrl}`, {
         signal: controllerRef.current?.signal,
         method: 'post',
         headers: {
           Authorization: `bearer ${token}`,
           'Content-Type': 'application/json',
         },
-        body: JSON.stringify({
-          blogUrl: values.blogUrl,
-        }),
+        body: JSON.stringify(urls),
```

Comment:
The response is `{feedUrl, type}` now so in line 178 you want to map this to an array of URL strings so React can render it.

Reply:
It's even better to add type to the response so TS can catch this early.

Reply:
What does "add type to the response" mean?

Reply:
I mean `const res: {feedUrls: [{feed:string, type: url}]} = await response.json();`

Reply:
Question: if I map this to extract the string URL values, we lose all the types (youtube, blog, twitch).  We probably need a larger fix here to maintain that data.  Should that happen in this PR, or a follow-up?  If this one, I need some guidance so it works with the formik stuff.

Reply:
The change for displaying the type can be in follow-up, for now, we need to map it to string because React can't display JSON

Reply:
I tried to make the change to line 178 like this:

```tsx
setFieldValue(
  'allFeeds',
  res.feedUrls.map((feed: { feedUrl: URL; type: string }) => {
    return feed.feedUrl;
  })
);
```
However I get a 404 when try to submit the data to the database.

Reply:
I've added the types and mapped the `feedUrl`.  We need to deal with the `type` in a follow up.